### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by pabgo**
This file builds a docker container based on Ubuntu 18.04 with Qt 5.12.0,CMake and other tools

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    && cd /
+
+# Install ODBC driver and ODBC driver manager
+
+RUN curl -sSL https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl -sSl https://packages.microsoft.com/config/ubuntu/18.04/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update \
+    && ACCEPT_EULA=Y apt-get -y install msodbcsql17 \
+    && ACCEPT_EULA=Y apt-get -y install mssql-tools \
+    #&& echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile \
```
Comment (wfallascr) [inline Dockerfile]: Clean up commented commands

**Review by wfallascr — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# Building ODBC plugin
+
+RUN set -x && cd /qt-everywhere-src-5.12.0/qtbase/src/plugins/sqldrivers \
+    && qmake -- ODBC_PREFIX=/usr/local/unixODBC \
+    && make sub-odbc \
+    && cd /
+
+# Clean compilation files
+
+#RUN cd $QT_BASE_DIR && make clean \
```
Comment (luvalverde) [inline Dockerfile]: why is it commented?

**Review by luvalverde — CHANGES_REQUESTED** [review]

**Comment by pabgo** [comment]
PR back to you @wfallascr @luvalverde

**Review by luvalverde — APPROVED** [review]
Looks ok for me, just squash your commits @pabgo

**Review by wfallascr — APPROVED** [review]

**Pull request merged**